filesystem/rootfs/docker/etc/docker.service

```diff
@@ -9,6 +9,7 @@ Type=notify
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
 ExecStart=/usr/bin/dockerd
+ExecStartPost=/usr/sbin/iptables -P FORWARD ACCEPT
```

Comment:
I am wondering if this will change a lot of firewall configuration. I just want to use sealer, while you change the whole configuration with the granted root access.

Reply:
Kernel module configs will be change when run the init script, If we don't do that, user should care too many configs, now we should have root access to reduce the complexity of use.